src/client/client.ts

```diff
@@ -1009,35 +1021,39 @@ export default class TaskClient {
       async () => {
         let ruConsumption = 0;
 
-        await Array.from(Object.values(sprocs)).map(async ({ id, fn }) => {
-          try {
-            const result = await this._client.replaceSproc(id, fn);
-            ruConsumption += result.ruConsumption || 0;
-          } catch (err) {
-            // 404 means we need to create it
-            if (IronTaskError.is(err, ErrorCode.DATABASE_RESOURCE_NOT_FOUND)) {
-              try {
-                const result = await this._client.createSproc(id, fn);
-                ruConsumption += result.ruConsumption || 0;
-              } catch (err) {
-                // 409s are okay at this point. Just means we
-                // were in a race with something else
-                if (
-                  IronTaskError.is(
-                    err,
-                    ErrorCode.DATABASE_RESOURCE_ALREADY_EXISTS
-                  )
-                ) {
-                  return;
+        void (await Array.from(Object.values(sprocs)).map(
```

Comment:
why the `void`?

Reply:
The new linter wants it here -- 
"""
An array of Promises may be unintentional. Consider handling the promises' fulfillment or rejection with Promise.all or similar, or explicitly marking the expression as ignored with the `void` operator
"""

I tried to make the fewest changes possible outside of what I was intentionally changing, let me know if a different approach would be better.

Reply:
ah, that's a good catch by the linter. this should be `Promise.all`ed instead of `void`ed